package.json

```diff
@@ -31,7 +31,7 @@
         "tsConfig": "<rootDir>/tsconfig.spec.json"
       }
     },
-    "rootDir": "__tests__"
+    "rootDir": "src/__tests__"
```

Comment:
can we remove custom rootDir at all? feel free to do it later

Reply:
We need it for `ts-jest` to find `tsconfig.spec.json`. We could also configure the location of the tsconfig it in `globals.ts-jest` in `package.json` instead.